pkg/aks/components/CruAks.vue

```diff
@@ -16,7 +16,7 @@ import CreateEditView from '@shell/mixins/create-edit-view';
 import FormValidation from '@shell/mixins/form-validation';
 import SelectCredential from '@shell/edit/provisioning.cattle.io.cluster/SelectCredential.vue';
 import CruResource from '@shell/components/CruResource.vue';
-import LabeledSelect from '@shell/components/form/LabeledSelect.vue';
+import LabeledSelect from '@shell/components/form/LabeledSelect/index.vue';
```

Comment:
does this need the index.vue? on other places it wasn't needed

Reply:
Same as https://github.com/rancher/dashboard/pull/10068#discussion_r1474680928